### PR TITLE
IIAB's main README

### PR DESCRIPTION
**Pull request by holta**
@aidan-fitz & @curtathompson can you help review this quick, to make sure it's all-around-inviting in useful ways to new people?

Many thanks if you can contribute an extra pair of eyes here (of course this is not yet a complete overhaul of http://internet-in-a-box.org and our long-awaited IIAB logo, but those WILL come in due course ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,35 @@
 # Internet-in-a-Box (IIAB)
 
-Welcome to the Git repository of the Internet-in-a-Box (IIAB) project. This is a community-based project developed and supported by volunteers from around the world. The Internet-in-a-Box (IIAB) is small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection. It provides a local content server of the world’s Free Knowledge.
-
-The older version of this repository is at [github.com/xsce](http://github.com/xsce). We are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
+Welcome to the Git repository for Internet-in-a-Box (IIAB).  This community-based project is enabled by professional volunteers around the world.  Internet-in-a-Box (IIAB) is a small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection.
```
Comment (veeara282) [inline README.md]: change to "including Wikipedia, OpenStreetMap, and Khan Academy" - the "like" and "and others" are redundant

File: README.md [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
 
-* [Ansible](http://www.ansible.com)
+#### Ansible IT Automation
+
+FYI we are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
 
 ## Contributing
 
-Please read our [Contributors Guide](https://github.com/iiab/iiab/wiki/IIAB-Contributors-Guide) to learn more about contributing to Internet-in-a-Box (IIAB) repository.
+We greatly welcome contributions from educators, librarians *and* IT/UX/QA people of all kind!
```
Comment (veeara282) [inline README.md]: "of all kinds"

File: README.md [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
-Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details about installation of Internet-in-a-Box (IIAB).
+Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details on installing of Internet-in-a-Box (IIAB).
+
+Or, to dive right in, our "1-line installer" is available off of http://download.iiab.io — it's generally best to click on the highest version number.
 
-## Built With
+Please see "What are the best places for community support?" at http://FAQ.IIAB.IO which many answers to help you along the way, in designing the digital knowledge hotspot most suitable for your own community.
 
-* [Ansible](http://www.ansible.com)
+#### Ansible IT Automation
```
Comment (veeara282) [inline README.md]: This should be a level 2 or 3 heading. I would change it back to "Built with Ansible".

File: README.md [hunk 4]
```diff
@@ -1,22 +1,35 @@
...
+FYI we are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
 
 ## Contributing
 
-Please read our [Contributors Guide](https://github.com/iiab/iiab/wiki/IIAB-Contributors-Guide) to learn more about contributing to Internet-in-a-Box (IIAB) repository.
+We greatly welcome contributions from educators, librarians *and* IT/UX/QA people of all kind!
+
+Please see "How can I help?" at http://FAQ.IIAB.IO
+
+Check out 2017's [Contributors Guide](https://github.com/iiab/iiab/wiki/IIAB-Contributors-Guide) to learn more about contributing directly to Internet-in-a-Box (IIAB) software and its open community architecture for education.
```
Comment (veeara282) [inline README.md]: Why "2017's"?

File: README.md [hunk 5]
```diff
@@ -1,22 +1,35 @@
...
+
+*Thank you for helping us enable offline access to the Internet's open knowledge jewels, as well as "Sneakernet-of-Alexandria" distribution of local/indigenous content, where mass media channels are not serving grassroots voices.*
```
Comment (veeara282) [inline README.md]: Why is this paragraph italicized?

File: README.md [hunk 6]
```diff
@@ -1,22 +1,35 @@
...
 
 ## Versioning
 
-You can track the latest Internet-in-a-Box (IIAB) [releases here](https://github.com/iiab/iiab/releases).
+Pre-releases of Internet-in-a-Box (IIAB) are available from http://download.iiab.io — click on the highest version number.
+
+You can also track the latest Internet-in-a-Box (IIAB) [official releases here](https://github.com/iiab/iiab/releases).
+
+Finally older versions are also available, from [github.com/xsce](http://github.com/xsce) and [schoolserver.org](http://schoolserver.org).
```
Comment (veeara282) [inline README.md:37]: This is all good!

**Review by veeara282 — CHANGES_REQUESTED** [review]
I typically make two passes through writing when I edit - my first pass consists of proofreading, and then I make my second pass through when I'm more familiar with the writing so I can deal with higher-level properties like style and content. This is my first pass - I'll make more stylistic edits when I have a chance.

File: README.md (hunk 5 above)
Comment (holta) [reply README.md]: Catalyzing grassroots education is important motivating force here, that mobilizes many to contribute.

(Or we can un-italicize later if this emphasis belongs elsewhere?)

File: README.md (hunk 1 above)
Comment (holta) [reply README.md]: Thanks @aidan-fitz !

All suggestions incorporated here: https://github.com/iiab/iiab/pull/1211/commits/32b133ac91184cb7cbbdf6ac79b601db259bafc1

File: README.md (hunk 3 above)
Comment (holta) [reply README.md]: FWIW level 2 and 3 heading fonts were tested earlier, and both were too big (in my opinion!)

**Comment by holta** [comment]
Merging for now, to the blurb underneath https://github.com/iiab/iiab will now be much more fresh.

Further contributions/suggestions welcome at anytime!

**Pull request merged**